pkg/osquery/runtime/osqueryinstance.go

```diff
@@ -802,6 +802,14 @@ func (i *OsqueryInstance) createOsquerydCommand(osquerydBinary string, paths *os
 	// https://github.com/osquery/osquery/pull/6824
 	cmd.Env = append(cmd.Env, "SYSTEM_VERSION_COMPAT=0")
 
+	// On Windows, we want the `SystemDrive` environment variable to be set to ensure paths can be resolved appropriately.
+	// The cmd handles setting `SystemRoot` for us.
+	if runtime.GOOS == "windows" {
+		if systemDrive, found := os.LookupEnv("SystemDrive"); found {
```

Comment:
I think we probably need to think hard about the `!found` case. It might be better to set this to `""` than to leave it unset.

Reply:
@directionless I tested and confirmed setting `SystemDrive=` helps, so I made that change in this follow-up PR: https://github.com/kolide/launcher/pull/1961